packages/gatsby-plugin-styled-components/README.md

```diff
@@ -27,6 +27,12 @@ module.exports = {
 }
 ```
 
+Edit `.babelrc`
+
+```
+plugins: [`babel-plugin-styled-components`]
```

Comment:
I found this change necessary on a site I was working on to resolve obscure and inconsistent styling issues.

Reply:
I think `gatsby-plugin-styled-components` does this automatically in https://github.com/gatsbyjs/gatsby/blob/bcd3bcccb321d911cd574939ec5a3ad763859d6d/packages/gatsby-plugin-styled-components/src/gatsby-node.js#L49-L53

Was the behavior `babel-plugin-styled-components` adds not getting picked up despite that? If so it might be a bigger issue we need to fix.

Did you happen to have a repro lying around? I can check it out